### PR TITLE
Add Bible plugin

### PR DESCRIPTION
**Pull request by CianLR**
Steely is good for all sorts, even Christian servers.

I plan to allow you to specify a specific verse but my internet is so bad rn I was barely able to get this out. So to be continued.

I wasn't sure whether or not to just commit the Bible file, I can do that if it's easier

### CODE REVIEW

**Review by iandioch — APPROVED** [review]
a few notes:
- There are some other plugins (eg. /emoji) that load an external file from a URL on startup if it can't be found on-disk. The emoji plugin is literally pulling a JSON file from some github repo too. I think that'd be a better approach here.
- nice use of `plugin.setup()`, if it can't find the file it should print a warning and disable the plugin. Did you try that? Does that bit work?
- This is a Protestant version of the bible. Please fix.

I'll LGTM anyway, sure. Good luck with your internet.

File: steely/plugins/bible/main.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import json
+import random
+
+from plugin import create_plugin, PluginManager
+from message import SteelyMessage
+
+HELP_STR = """
+Request your faviourite bible quotes, right to the chat.
```
Comment (iandioch) [inline steely/plugins/bible/main.py:8]: favourite*

File: steely/plugins/bible/main.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import json
+import random
+
+from plugin import create_plugin, PluginManager
```
Comment (iandioch) [inline steely/plugins/bible/main.py:4]: `PluginManager` doesn't need to be imported.

(I realise it's also imported in `/nw`, I'll remove it at some stage)

**Comment by byxor** [comment]
These new plugin annotations are sick. Definitely gonna share this with my church group on Sunday.

File: steely/plugins/bible/main.py (hunk 2 above)
Comment (CianLR) [reply steely/plugins/bible/main.py]: My copy and pasting has been revealed.

**Comment by CianLR** [comment]
Thx for the review, I'll definitely tackle your notes. My source didn't have a good old Catholic bible unfortunately tho

**Pull request merged**